api/src/opentrons/protocol_engine/state/labware.py

```diff
@@ -248,6 +259,17 @@ def get_id_by_module(self, module_id: str) -> str:
             "There is no labware loaded on this Module"
         )
 
+    def raise_if_labware_is_not_on_top(self, labware_id: str) -> None:
```

Comment:
can we change this method name? I find this very confusing. maybe we can do something like:
```suggestion
    def raise_if_labware_has_labware_on_top(self, labware_id: str) -> None:
```

really hard finding names for this PR ;-)

Reply:
```suggestion
    def raise_if_labware_has_labware_above(self, labware_id: str) -> None:
```

!!!